egs/wsj/s5/utils/data/perturb_data_dir_volume.sh

```diff
@@ -43,6 +43,26 @@ fi
 # foo-1 cat foo.wav | sox --vol 1.6 -t wav - -t wav - |    # volume perturbation done
 # bar-1 sox --vol 1.2 bar.wav -t wav - |                   # volume perturbation done
 # foo-2 wav-reverberate --additive-signals="sox --vol=0.1 noise1.wav -t wav -|" foo.wav |   # volume perturbation not done
+# volume_perturb_done=`head -n100 $data/wav.scp | python -c "
```

Comment:
thanks.  You can remove this commented block.

Reply:
I have removed the commented block.